server/embedly.js

```diff
@@ -17,6 +17,12 @@ var logFactory = require('./logger');
 var log = logFactory('server.services.embedly');
 var npmLog = logFactory('server.services.embedly.vendor');
 
+// returns hex number without the prepended '#'
+var _rgbToHex = function(r, g, b) {
+  /*jshint bitwise: false */
+  return ((1 << 24) + (r << 16) + (g << 8) + b).toString(16).slice(1);
+};
```

Comment:
Would it be any easier to just accept an array of 3 items/colors?

```js
// returns hex number without the prepended '#'
var _rgbToHex = function(arr) {
  /*jshint bitwise: false */
  return ((1 << 24) + (arr[0] << 16) + (arr[1] << 8) + arr[2]).toString(16).slice(1);
};
```

Then you can simplify the caller by just doin':
```js
out.extractedFaviconColor = _rgbToHex(d.favicon_colors[0].color);
```

And then not have to deal with doing this:
```js
+          r = d.favicon_colors[0].color[0];
+          g = d.favicon_colors[0].color[1];
+          b = d.favicon_colors[0].color[2];
```